.eslintignore

```diff
@@ -0,0 +1 @@
+dist/*
```

Comment:
Did you forget to add `node_modules` and build bundles?

Reply:
@danyloburavlov, ESLint documentation says that `node_modules/` and different dot files are implicitly ignored by default. `dist/` folder which contains build bundles is already added to `.eslintignore`.